util/config_file.c

```diff
@@ -1448,6 +1453,41 @@ create_cfg_parser(struct config_file* cfg, char* filename, const char* chroot)
 	init_cfg_parse();
 }
 
+void
+config_auto_slab_values(struct config_file* cfg)
+{
+#define SET_AUTO_SLAB(var, name, val)					\
+do {									\
+	if(cfg->var == 0) {						\
+		cfg->var = val;						\
+		verbose(VERB_QUERY, "setting "name": %lu", val);	\
```

Comment:
Cast printout val to long.
```suggestion
		verbose(VERB_QUERY, "setting "name": %lu", (unsigned long)val);	\
```